src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import './UserInfo.scss';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+interface Props {
+  user: User | null;
```

Comment:
`UserInfo` should know nothing of `user` being possible null, just use conditional rendering on one level up